weatherAplication/Modules/Home/View/HomeViewController.swift

```diff
@@ -81,10 +96,41 @@ class HomeViewController: UIViewController {
 
 }
 
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let lastCity = viewModel.lastSearches
+        return lastCity.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath as IndexPath)
+        cell.textLabel?.textColor = .yellow
+        cell.backgroundColor = .blue
+        let lastCityArray = viewModel.lastSearches
+        cell.textLabel?.text = lastCityArray[indexPath.row].city
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let cityInfoToSend = viewModel.lastSearches[indexPath.row]
+        showWeather(cityInfo: cityInfoToSend)
```

Comment:
Informacja o kliknięciu w cellkę powinna zostać przekazana do viewModelu i to on powinien decydować co z tym zrobić